tutorial/practicals/LFRic/distributed_memory/3_overlapping_comms/solutions/overlap_generic.py

```diff
@@ -76,7 +76,7 @@ def trans(psy):
         # Get the schedule (the PSyIR representation of the PSy-layer)
         schedule = invoke.schedule
         # Split any synchronous halo exchanges into asynchronous halo exchanges
-        for hex_node in schedule.walk(DynHaloExchange):
+        for hex_node in schedule.walk(LFRicHaloExchange):
```

Comment:
Please add yourself under "Modified", i.e.
```
# Author: R. W. Ford, STFC Daresbury Lab
# Modified: O. Brunt, Met Office
```

Please also update the modification year to 2023.